Symfony/CS/Fixer/PSR2/SwitchCaseSpaceFixer.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\PSR2;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class SwitchCaseSpaceFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        foreach ($tokens as $index => $token) {
+            if (!$token->isGivenKind(array(T_CASE, T_DEFAULT))) {
+                continue;
+            }
+
+            $ternariesCount = 0;
+            for ($colonIndex = $index + 1;; ++$colonIndex) {
+                // We have to skip ternary case for colons.
+                if ($tokens[$colonIndex]->equals('?')) {
+                    ++$ternariesCount;
+                }
+
+                if ($tokens[$colonIndex]->equals(':') && 0 === $ternariesCount) {
+                    break;
+                }
+
+                if ($tokens[$colonIndex]->equals(':')) {
+                    --$ternariesCount;
+                }
+            }
+
+            $valueIndex = $tokens->getPrevNonWhitespace($colonIndex);
+
+            if ($colonIndex - $valueIndex > 1) {
+                $tokens->clearRange($valueIndex + 1, $colonIndex - 1);
```

Comment:
I'm not talking about the style here. The problem is that in all the test cases you have provided, in run time the first argument is exactly the same as second one.
So you could simply wrote `$tokens[$valueIndex + 1]->clear();` and have the same effect.

I bet it is not what you want and just need to extend the tests